phpcs.xml.dist

```diff
@@ -11,7 +11,7 @@
 
     <rule ref="Libero"/>
 
-    <file>src/</file>
+    <file>Libero/</file>
```

Comment:
To avoid the namespace being `Libero\CodingStandard\Libero`. (Is there a way to have PHP_CodeSniffer work without having the ruleset's parent directory being the name? It's already defined in the ruleset itself...)

Reply:
Hmm, think we'll have to revert this. The sniff name ends up being `CodingStandard. WhiteSpace.ObjectStaticOperatorSpacing` rather than the expected `Libero. WhiteSpace.ObjectStaticOperatorSpacing`.

Reply:
What's the decision here in the end? Happy either way